app/models/maintenance_tasks/runner.rb

```diff
@@ -39,8 +40,12 @@ def initialize(run)
     def run(name:, csv_file: nil)
       run = Run.active.find_by(task_name: name) || Run.new(task_name: name)
 
-      if Task.named(name) < CsvCollection && csv_file
-        run.csv_file.attach(csv_file)
+      if Task.named(name) < CsvCollection
+        if csv_file
+          run.csv_file.attach(csv_file)
+        else
+          raise "#{name} must be supplied with a CSV file to process."
```

Comment:
I think we should make it a ValidationError from a Run missing the CSV for a CSV task.